src/Ledger/Address.lagda

```diff
@@ -1,4 +1,7 @@
 \section{Addresses}
+\label{sec:addresses}
+\modulenote{\LedgerAddress}
```

Comment:
I think it will be very helpful to readers to know exactly where they can find the full, unabridged Agda module from which the code in each section is extracted.  I made a macro called `\modulenote{}` (at the bottom of the `preamble.tex` file), so we can include the same information at the start of each section by passing `\modulenote{}` the appropriate Agda module macro for that section.

Reply:
Makes total sense.